packages/api/models/nomination.js

```diff
@@ -140,6 +140,14 @@ module.exports = (sequelize, DataTypes) => {
         allowNull: true,
         type: DataTypes.DATE
       },
+      awaitingHipaaTimestamp: {
+        allowNull: true,
+        type: DataTypes.DATE
+      },
+      reminderSent: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: false
```

Comment:
Since we may want to record when this email was sent, would it be better to use a date here?

Reply:
Talked to product about this. We already have the timestamps for when an application enters a stage, then this flag is set to False, if an email reminder is required, then this is set to True so that we do not send more than 1 reminder per app, per stage. (Although right now I believe only two stages require reminder emails)

Reply:
Actually I will add the fields, as the timestamps that currently exist are for when the nomination enters that field, not when the email is sent.